db/schema.rb

```diff
@@ -10,14 +10,14 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2018_07_13_083524) do
+ActiveRecord::Schema.define(version: 2018_07_17_095526) do
 
   create_table "comments", force: :cascade do |t|
-    t.string "author"
     t.text "body"
     t.integer "post_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer "user_id"
```

Comment:
how about index?

Reply:
indexes added user_id in both posts and comments tables